applications/schema/schema/application.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "YAML Application Description",
+  "description": "A full description of a YAML application AICA framework.",
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
I would remove this, especially since we are missing some fields like events. Even if we have a complete description, I don't see a reason to prohibit metadata or other fields.